.circleci/config.yml

```diff
@@ -1,12 +1,13 @@
 version: 2
 jobs:
-  build:
+  test-frontend-and-server:
     docker:
-      - image: cypress/browsers:chrome67
+      #      - image: cypress/browsers:chrome67
+      - image: circleci/python:3.6.7-node-browsers
     steps:
       - checkout
       - restore_cache: # special step to restore the dependency cache
-          key: dotcache-cache-{{ checksum "package.json" }}
+          key: dotcache-cache-v20190430-{{ checksum "package.json" }}
```

Comment:
Curious: Why the hard-coded version here and below?

Also: When do we update this hard-coded version?

Reply:
had to update it cause image change required it. I thought date would be a good way to tag it. I can add a comment?

Reply:
Ideally solution ✨ automagical 🦄 caching. 

There are many things we can use to create a unique key:
- [Keys and Templates](https://circleci.com/docs/2.0/caching/#using-keys-and-templates)
- [Built-in Environment Variables](https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables)

My suggestion:
```
key: bmd-cache-{{checksum ".circleci/config.yml"}}-{{ checksum "package.json" }}
```

Reply:
Tracking here: https://github.com/votingworks/bmd/issues/285